### PR TITLE
mark Safari 10 as stable, update Safari TP to 13

### PR DESCRIPTION
**Pull request by chicoxyzzy**
this PR also includes fix for https://github.com/kangax/compat-table/issues/910

### CODE REVIEW

**Comment by kangax** [comment]
@chicoxyzzy hey, just added you to the repo! thanks for all the help :)

File: data-es6.js [hunk 1]
```diff
@@ -508,19 +508,20 @@ exports.browsers = {
   safari71_8: {
     full: 'Safari',
     short: 'SF 7.1,<br>SF 8',
-    obsolete: false
+    obsolete: true
   },
   safari9: {
     full: 'Safari',
     short: 'SF 9',
+    obsolete: false,
```
Comment (afmenez) [inline data-es6.js:516]: If Safari 10 was released, shouldn't Safari 9 be obsolete now?

File: data-es5.js [hunk 2]
```diff
@@ -121,9 +121,15 @@ exports.browsers = {
...
     family: 'JavaScriptCore',
-    short: 'SF 6-9',
+    short: 'SF 6-8',
+    obsolete: true
+  },
+  safari9: {
+    full: 'Safari 9',
+    family: 'JavaScriptCore',
+    short: 'SF 9',
     obsolete: false
```
Comment (afmenez) [inline data-es5.js]: If Safari 10 was released, shouldn't Safari 9 be obsolete now?

File: data-esintl.js [hunk 3]
```diff
@@ -113,18 +113,28 @@ exports.browsers = {
...
     full: 'Safari',
-    short: 'SF 7.1,<br>SF 8',
-    obsolete: false
+    short: 'SF 7.1-8',
+    obsolete: true,
+  },
+  safari9: {
+    full: 'Safari',
+    short: 'SF 9',
+    obsolete: false,
```
Comment (afmenez) [inline data-esintl.js]: If Safari 10 was released, shouldn't Safari 9 be obsolete now?

**Comment by mkurz** [comment]
iOS10 is stable now as well

File: data-es6.js (hunk 1 above)
Comment (chicoxyzzy) [reply data-es6.js:516]: We mark 2 latest versions of Safari as stable. This line was added just to make it easier to change it to `true` later

**Comment by chicoxyzzy** [comment]
@mkurz that's true. We should update all iOS in all tabs. Mobile Safari data is outdated everywhere and it's a good idea for another PR. It should be very easy. All iOS data should correspond current Safari data. Do you want to take a look? :)

**Pull request merged**